deps/llvm.mk

```diff
@@ -517,6 +517,20 @@ $(eval $(call LLVM_PROJ_PATCH,llvm-12-fde-symbols-aarch64)) # remove for LLVM 13
 $(eval $(call LLVM_PROJ_PATCH,llvm-12-force-eh_frame-aarch64)) # remove for LLVM 13
 endif # LLVM_VER 12.0
 
+ifeq ($(LLVM_VER_SHORT),13.0)
+$(eval $(call LLVM_PATCH,llvm7-revert-D44485)) # Needs upstreaming
+$(eval $(call LLVM_PROJ_PATCH,llvm-13-D75072-SCEV-add-type))
+$(eval $(call LLVM_PATCH,llvm-julia-tsan-custom-as))
```

Comment:
Do we want to drop this for now, and instead revert the ptrtoint/inttoptr change so that we get errors if this is needed?

Reply:
Fat fingered on my phone. I meant the patch one line higher.

Reply:
it is likely still needed (and there is a likely LLVM reviewer now), it just needs to be generalized to all pointers, not just NI